addHandling.php

```diff
@@ -0,0 +1,103 @@
+<?php
+require_once 'src/Models/CarsModel.php';
+require_once 'src/Models/MakeModel.php';
+$allFieldsFilled = true;
+
+
+if (!isset($_POST['model']) || empty($_POST['model'])) 
+{
+    $allFieldsFilled = false;
+}
+
+if (!isset($_POST['make']) || empty($_POST['make'])) 
+{
+    $allFieldsFilled = false;
+}
+
+if (!isset($_POST['bodytype']) || empty($_POST['bodytype'])) 
+{
+    $allFieldsFilled = false;
+}
+
+if (!isset($_POST['year']) || empty($_POST['year'])) 
+{
+    $allFieldsFilled = false;
+}
+
+if (!isset($_POST['image_link']) || empty($_POST['image_link'])) 
+{
+    $allFieldsFilled = false;
+}
+
+if (!$allFieldsFilled) 
+{
+    header('Location: add.php?error=fields not filled');
+    exit();
+}
+
+$model = $_POST['model'];
+$make = $_POST['make'];
+$bodytype = $_POST['bodytype'];
+$year = $_POST['year'];
+$image_link = $_POST['image_link'];
+
+if (strlen($model) > 100)
+{
+    header('Location: add.php?error=model name too long');
+    exit();
+}
+elseif (strlen($image_link) > 500)
+{
+    header('Location: error.php?error=image link too long');
+    exit();
+}
+elseif ($year < 1900 || $year > 2024)
```

Comment:
Instead of hard coding 2024, I would suggest you do something like:  
 

```
date("Y") + 1;
```

So it's always 1 year a head, you don't have to remember to update it each year